cmake/Modules/Findspdlog.cmake

```diff
@@ -1,27 +1,45 @@
 add_library(spdlog INTERFACE IMPORTED)
 
-find_path(spdlog_INCLUDE_DIR spdlog/spdlog.h)
+unset(spdlog_INCLUDE_DIR CACHE)
```

Comment:
Why is is required for spdlog, and not for fmt?
>If the file in a directory is found the result is stored in the variable and the search will not be repeated unless the variable is cleared. If nothing is found, the result will be <VAR>-NOTFOUND, and the search will be attempted again the next time find_path is invoked with the same variable.

Please explain why is it required.

Reply:
It was left unintentionally after debugging.